src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -4342,16 +4355,21 @@ public partial class TypeUnloadedException : System.SystemException
         public static bool TryParse(string? s, out System.UInt64 result) { throw null; }
     }
     [System.CLSCompliantAttribute(false)]
-    public readonly partial struct UIntPtr : System.IEquatable<System.UIntPtr>, System.Runtime.Serialization.ISerializable
+    public readonly partial struct UIntPtr : System.IComparable, System.IComparable<System.UIntPtr>, System.IEquatable<System.UIntPtr>, System.IFormattable, System.Runtime.Serialization.ISerializable
     {
         private readonly int _dummyPrimitive;
         public static readonly System.UIntPtr Zero;
         public UIntPtr(uint value) { throw null; }
         public UIntPtr(ulong value) { throw null; }
         public unsafe UIntPtr(void* value) { throw null; }
+        public static System.UIntPtr MaxValue { get { throw null; } }
+        public static System.UIntPtr MinValue { get { throw null; } }
         public static int Size { get { throw null; } }
```

Comment:
I just noticed that `ShiftMask` is missing. These should be exposed as part of the PR and should be properties as per: https://github.com/dotnet/runtime/issues/21943#issuecomment-555895614

Reply:
The C# compiler will emit the equivalent code, `(sizeof(nint) * 8) - 1`, explicitly rather than using the `ShiftMask` members, so the members are not needed for the compiler.

---
In reply to: [408345037](https://github.com/dotnet/runtime/pull/307#discussion_r408345037) [](ancestors = 408345037)

Reply:
Thanks @cston. We shouldn't need to expose that member if the compiler won't actually use it. CC. @terrajobst, @dotnet/fxdc